packages/livecd/grub2/config/grub_live.cfg

```diff
@@ -21,9 +21,9 @@ if [ -f ${font} ];then
 fi
 menuentry "cOS" --class os --unrestricted {
     echo Loading kernel...
-    $linux ($root)/boot/kernel.xz cdroot root=live:CDLABEL=COS_LIVE rd.live.dir=/ rd.live.squashimg=rootfs.squashfs console=tty1 console=ttyS0 rd.cos.disable
+    $linux ($root)/boot/kernel cdroot root=live:CDLABEL=COS_LIVE rd.live.dir=/ rd.live.squashimg=rootfs.squashfs console=tty1 console=ttyS0 rd.cos.disable
     echo Loading initrd...
-    $initrd ($root)/boot/rootfs.xz
+    $initrd ($root)/boot/initrd
```

Comment:
This was a long standing TODO we had in elemental-cli. These paths used to be misleading, there was nothing forcing or telling us to use `xz` compression, this is an arbitrary name set by us and we are not forcing `xz` in any way. It is likely to be the default for SLE15-SP4, but, for instance it is not for TW, there dracut defaults to `zst`.